legend-engine-language-pure-grammar-api/src/test/java/org/finos/legend/engine/language/pure/grammar/api/test/TestGrammarToJsonApi.java

```diff
@@ -90,13 +90,13 @@ public void testMixedParsingErrors()
     public void testMappingIncludeSerialization()
     {
         // Mapping include should yield `includedMapping` instead of `includedMappingPackage` and `includedMappingName`
-        test("{\"code\": \"###Mapping\\nMapping A\\n(\\n include test::B\\n)\"}", "{\"isolatedLambdas\":{\"lambdas\":{}},\"modelDataContext\":{\"_type\":\"data\",\"elements\":[{\"_type\":\"mapping\",\"associationMappings\":[],\"classMappings\":[],\"enumerationMappings\":[],\"includedMappings\":[{\"includedMapping\":\"test::B\",\"sourceInformation\":{\"endColumn\":16,\"endLine\":4,\"sourceId\":\"\",\"startColumn\":2,\"startLine\":4}}],\"name\":\"A\",\"package\":\"\",\"sourceInformation\":{\"endColumn\":1,\"endLine\":5,\"sourceId\":\"\",\"startColumn\":1,\"startLine\":2},\"tests\":[]},{\"_type\":\"sectionIndex\",\"name\":\"SectionIndex\",\"package\":\"__internal__\",\"sections\":[{\"_type\":\"importAware\",\"elements\":[],\"imports\":[],\"parserName\":\"Pure\",\"sourceInformation\":{\"endColumn\":8,\"endLine\":1,\"sourceId\":\"\",\"startColumn\":1,\"startLine\":1}},{\"_type\":\"importAware\",\"elements\":[\"A\"],\"imports\":[],\"parserName\":\"Mapping\",\"sourceInformation\":{\"endColumn\":1,\"endLine\":7,\"sourceId\":\"\",\"startColumn\":8,\"startLine\":2}}]}]},\"renderStyle\":\"STANDARD\"}");
+        test("{\"code\": \"###Mapping\\nMapping A\\n(\\n include test::B\\n)\"}", "{\"isolatedLambdas\":{\"lambdas\":{}},\"modelDataContext\":{\"_type\":\"data\",\"elements\":[{\"_type\":\"mapping\",\"associationMappings\":[],\"classMappings\":[],\"enumerationMappings\":[],\"includedMappings\":[{\"includedMapping\":\"test::B\",\"sourceInformation\":{\"endColumn\":16,\"endLine\":4,\"sourceId\":\"\",\"startColumn\":2,\"startLine\":4}}],\"name\":\"A\",\"package\":\"\",\"sourceInformation\":{\"endColumn\":1,\"endLine\":5,\"sourceId\":\"\",\"startColumn\":1,\"startLine\":2},\"testSuites\":[],\"tests\":[]},{\"_type\":\"sectionIndex\",\"name\":\"SectionIndex\",\"package\":\"__internal__\",\"sections\":[{\"_type\":\"importAware\",\"elements\":[],\"imports\":[],\"parserName\":\"Pure\",\"sourceInformation\":{\"endColumn\":8,\"endLine\":1,\"sourceId\":\"\",\"startColumn\":1,\"startLine\":1}},{\"_type\":\"importAware\",\"elements\":[\"A\"],\"imports\":[],\"parserName\":\"Mapping\",\"sourceInformation\":{\"endColumn\":1,\"endLine\":7,\"sourceId\":\"\",\"startColumn\":8,\"startLine\":2}}]}]},\"renderStyle\":\"STANDARD\"}");
```

Comment:
why are these changed?

Reply:
testSuites is now an empty array as well

Reply:
Resolved